packages/terra-tabs/src/common-tabs/_Tabs.module.scss

```diff
@@ -21,4 +22,13 @@
       visibility: hidden;
     }
   }
+
+  .container {
+    display: -webkit-box; // stylelint-disable value-no-vendor-prefix
+  }
+
+  .divcontainer {
+    max-width: 40px;
```

Comment:
How did we arrived at value 40px as Max-Width property ..??

Reply:
This divcontainer contains only AddButton icon ,this maximum width has been given to achieve the responsive behaviour .

Reply:
Does this Max-width is tested on all the terra supported viewports ..??

Reply:
Yes, this is tested in all the viewports.